proposals/simd/simdsignandunsignedextendedloads.md

```diff
@@ -0,0 +1,64 @@
+### **Proposal WebAssembly SIMD Modification**
+
+Currently as proposed there is an instructions defined in the WASM SIMD ISA as follows.
+
+**i8x16.mul** which is a register to register operation that takes 16 8 bit integers and
+
+multiplies them together resulting in an 8 bit value. If the distribution of the integers it flat this
+
+would result in a large percent of the instructions with overflow. This is a problem for many applications.
+
+
+### Proposed new instructions
+
+Six new load instructions are being proposed to make integer multiplies easier. i8x16zxload, i8x16sxload, i16x8zxload, i16x8sxload, i32x4zxload, i32x4sxload. This would make i8, i16, i32 multiplies useful and more practical for applications such as machine learning, image compression and video and rendering data processing.The new instructions would take consecutive integers of the corresponding size and zero sign extend and sign extend the consecutive bytes, words or dword to the promoted size of signed or unsigned data. An example of zero sign extend is shown below:  Intel and ARM both have this capability by doing the following:
```

Comment:
Nowhere does the text say what the proposed instructions actually do, what are their argument types, results, operational semantics, etc.